app/views/submitted_proposals/_outcome_location_window.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="modal fade" id="outcome-window" tabindex="-1" aria-labelledby="outcome-windowLabel" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title">Outcome Location</h5>
+        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+      </div>
+      <div class="modal-body text-justify" style="text-align: justify;" >
+        <div class="mb-3">
+          <div class="mb-3">
+            <label>Outcome</label>
+            <%= select_tag :outcome, options_for_select([['Approved', 'Approved'], ['Rejected', 'Rejected']]), include_blank: '', class: 'form-select w-25', data: { 'submitted-proposals-target': 'outcome' } %>
+          </div>
+          <div class="mb-3">
+            <label>Location</label>
+            <%= select_tag :location, options_for_select(locations), include_blank: '', class: 'form-select w-25', data: { 'submitted-proposals-target': 'selectedLocation' } %>
+          </div>
+          <div class="mb-3">
+            <label class="form-label">Size</label>
+             <!-- # TODO pending proposal assigned size functionality  -->
+          </div>
```

Comment:
You could just copy [this file](https://github.com/birs-math/proposals/blob/3061fb51007666c63004417bc77cb540ba4fbb56/db/migrate/20211125001539_add_assigned_size_to_proposals.rb) into your PR. :)